getopt returns an int.
on non-x86 architectures where char is unsigned, this breaks starting the
compositor
(i think)
diff --git a/main.c b/main.c
index 50288b0..a6b03ed 100644
--- a/main.c
+++ b/main.c
@@ -144,7 +144,7 @@ parse_options(int argc, char **argv, struct options *options)
   char *config_path = NULL;
   char *autostart = NULL;
 
-  char flag;
+  int flag;
   while ((flag = getopt(argc, argv, "vhc:a:")) != -1) {
     switch (flag) {
       case 'a':
